index.js

```diff
@@ -0,0 +1,52 @@
+var DAYS = [ 'SUN','MON', 'TUE', 'WED','THU', 'FRI', 'SAT'];
+var MONTHS = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'];
+
+var calendarDates = document.querySelectorAll('tbody td');
+var currentDay = document.querySelectorAll('.current-day p');
+
+var d = new Date();
+var nowYear = d.getFullYear();
+var nowMonth = d.getMonth();
+var nowDate = d.getDate();
+var nowDay = d.getDay();
+var firstDay = new Date(nowYear, nowMonth, 1);
+var lastDay = new Date(nowYear, nowMonth + 1, 0);
+var markToday;
+
+function addZero(num) {
+    if(num < 10) {
+        return ('0' + num);
+    }
+    else {
+        return num;
+    }
+}
```

Comment:
```suggestion
function addZero(num) {
    if (num < 10) {
        return '0' + num;
    } else {
        return num;
    }
}
```
if-else 문의 경우, 통상적으로 아래와 같은 코드 컨벤션을 따릅니다.

```js
if (test) {
  thing1();
  thing2();
} else {
  thing3();
}
```

또한 for, if 문과 같은 조건문과 반복문의 경우 여는 괄호 앞에 공백을 추가해주세요.

- https://github.com/airbnb/javascript#blocks--cuddled-elses
- [https://github.com/airbnb/javascript#whitespace--around-keywords](https://github.com/airbnb/javascript#whitespace--around-keywords)